plugins/GSdx/GSdx.rc

```diff
@@ -213,31 +214,32 @@ BEGIN
     LTEXT           "CRC Hack Level:",IDC_CRC_LEVEL_TEXT,22,208,79,8
     COMBOBOX        IDC_CRC_LEVEL,105,206,127,63,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     CONTROL         "Enable HW Hacks",IDC_HACKS_ENABLED,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,11,223,71,10
-    PUSHBUTTON      "Configure Hacks...",IDC_HACKSBUTTON,105,221,127,14
+    PUSHBUTTON      "Configure Hacks",IDC_HACKSBUTTON,105,221,127,14
```

Comment:
I remove the ... stuffs since it's just a waste of space, can't really post the screenshots right now since I did these changes on Git online editor through my mobile. would be nice if you could double check the recent changes :)